Kernel failed to build: undefined reference to zlib deflation functions

```
  LD      .tmp_vmlinux1
/nix/store/wj7hxrvf8jqipkdry7x6ii1ds1ry2hd5-binutils-2.43.1/bin/ld: warning: -z relro ignored
/nix/store/wj7hxrvf8jqipkdry7x6ii1ds1ry2hd5-binutils-2.43.1/bin/ld: Unexpected GOT/PLT entries detected!
/nix/store/wj7hxrvf8jqipkdry7x6ii1ds1ry2hd5-binutils-2.43.1/bin/ld: Unexpected run-time procedure linkages detected!
/nix/store/wj7hxrvf8jqipkdry7x6ii1ds1ry2hd5-binutils-2.43.1/bin/ld: drivers/gpu/drm/drm_panic.o: in function `drm_panic_qr_init':
/build/source/build/../drivers/gpu/drm/drm_panic.c:678:(.init.text+0x9c): undefined reference to `zlib_deflate_workspacesize'
make[2]: *** [../scripts/Makefile.vmlinux:34: vmlinux] Error 1
make[1]: *** [/build/source/Makefile:1166: vmlinux] Error 2
make: *** [../Makefile:224: __sub-make] Error 2
```

Got this bizzare error. Flake is on https://github.com/NixOS/nixpkgs/commit/4c2fcb090b1f3e5b47eaa7bd33913b574a11e0a0 .

--- a/packages/x1e80100-linux.nix
+++ b/packages/x1e80100-linux.nix
@@ -4,10 +4,10 @@ linuxPackagesFor (buildLinux {
   src = fetchFromGitHub {
     owner = "jhovold";
     repo = "linux";
-    rev = "wip/x1e80100-6.12-rc1";
-    hash = "sha256-PzE3Ittt787Lxl84Btz83AcE/LCN3IONhZKGZG34Qm8=";
+    rev = "wip/x1e80100-6.12-rc4";
+    hash = "sha256-lW9NjL4HQzeQpPnBVpUr3m5ZKV+f8jWC77i5en9IgOo=";
   };
-  version = "6.12.0-rc1";
+  version = "6.12.0-rc4";
   defconfig = "johan_defconfig";
 
   structuredExtraConfig = with lib.kernel; {

